instrumentation/opentelemetry-instrumentation-asyncio/src/opentelemetry/instrumentation/asyncio/__init__.py

```diff
@@ -0,0 +1,378 @@
+------------
+.. code:: python
+
+    # export OTEL_PYTHON_ASYNCIO_FUTURE_TRACE_ENABLED=true
+
+    loop = asyncio.get_event_loop()
+
+    future = asyncio.Future()
+    future.set_result(1)
+    task = asyncio.ensure_future(future)
+    loop.run_until_complete(task)
+
+3. to_thread
+-------------
+.. code:: python
+
+    import asyncio
+    from opentelemetry.instrumentation.asyncio import AsyncioInstrumentor
+
+    AsyncioInstrumentor().instrument()
+
+    async def main():
+        await asyncio.to_thread(func)
+
+    def func():
+        pass
+
+    asyncio.run(main())
+
+
+asyncio metric types
+---------------------
+
+* asyncio.process.duration (seconds) - Duration of asyncio process
+* asyncio.process.count (count) - Number of asyncio process
+
+
+API
+---
+"""
+import asyncio
+import sys
+from asyncio import futures
+from timeit import default_timer
+from typing import Collection
+
+from wrapt import wrap_function_wrapper as _wrap
+
+from opentelemetry.instrumentation.asyncio.package import _instruments
+from opentelemetry.instrumentation.asyncio.utils import (
+    get_coros_to_trace,
+    get_future_trace_enabled,
+    get_to_thread_to_trace,
+)
+from opentelemetry.instrumentation.asyncio.version import __version__
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.utils import unwrap
+from opentelemetry.metrics import get_meter
+from opentelemetry.trace import get_tracer
+from opentelemetry.trace.status import Status, StatusCode
+
+ASYNCIO_PREFIX = "asyncio."
+
+
+class AsyncioInstrumentor(BaseInstrumentor):
+    """
+    An instrumentor for asyncio
+
+    See `BaseInstrumentor`
+    """
+
+    methods_with_coroutine = [
+        "create_task",
+        "ensure_future",
+        "wait_for",
+        "wait",
+        "as_completed",
+        "run_coroutine_threadsafe",
+    ]
+
+    def __init__(self):
+        super().__init__()
+        self.process_duration_metric = None
+        self.process_counts_metric = None
+
+        self._tracer = None
+        self._meter = None
+        self._coros_name_to_trace: set = set()
+        self._to_thread_name_to_trace: set = set()
+        self._future_active_enabled: bool = False
+
+    def instrumentation_dependencies(self) -> Collection[str]:
+        return _instruments
+
+    def _instrument(self, **kwargs):
+        tracer_provider = kwargs.get("tracer_provider")
+        self._tracer = get_tracer(__name__, __version__, tracer_provider)
+        self._meter = get_meter(
+            __name__, __version__, kwargs.get("meter_provider")
+        )
+
+        self._coros_name_to_trace = get_coros_to_trace()
+        self._future_active_enabled = get_future_trace_enabled()
+        self._to_thread_name_to_trace = get_to_thread_to_trace()
+
+        self.process_duration_metric = self._meter.create_histogram(
+            name="asyncio.process.duration",
+            description="Duration of asyncio process",
+            unit="seconds",
```

Comment:
Please check [this](https://github.com/open-telemetry/semantic-conventions/blob/main/model/metrics/system-metrics.yaml#L49).

Reply:
I changed it to "s". Thank you